react-native/features/deep-linking.md

```diff
@@ -0,0 +1,61 @@
+# [MO] How to implement deep linking with react-navigation
+
+## Owner : [Nicolas Ngomai](https://github.com/lechinoix)
+
+* What is deep linking ?
+
+Deeplinking permits you to register a custom URL Scheme in your phone so that when calling a url as `myapp://mypath?param=value` the phone will open the app with the `mypath?param=value`.
+In React Native, you can use the Linking Module that handle the external links in your app.
+When calling a deeplink in your app, a 'url' event will be dispatch with the url associated.
+
+## How to implement ?
+
+- Follow this guide from react navigation : https://reactnavigation.org/docs/guides/linking
```

Comment:
Uhm doing this link is good, but it can even better if you can redo it like we did together, with check that it works every 5 min max.

You can check the graphQL tutorial for it.